lib/app/Readme.md

```diff
@@ -8,12 +8,20 @@ In your configuration, you can enable an app using something like this:
 
 ```yaml
 apps:
-    # Replace appName with the name of the app you wish to enable
-    appName:
-        responseCache: 180
-        disabled: false
+  # Replace appName with the name of the app you wish to enable
+  appName:
+    responseCache: 180
+    disabled: false
+    sharedData:
```

Comment:
TBH I am not sure why this name  is good, however; we are not really sharing data, but simply passing additional configuration to user-land, if I get this correctly.